specification/cognitiveservices/OpenAI.Inference/routes/audio_transcription.tsp

```diff
@@ -28,7 +28,7 @@ namespace Azure.OpenAI;
 @sharedRoute
 op getAudioTranscriptionAsPlainText is Azure.Core.ResourceAction<
   Deployment,
-  AudioTranscriptionOptions,  // response_format must be one of: text, srt, vtt
+  BodyParameter<AudioTranscriptionOptions>,  // response_format must be one of: text, srt, vtt
```

Comment:
I don't quite understand the need for the comment (neither before nor after :))

Reply:
i can go ahead and remove it, I honestly didn't even look at it, if there's a doc it should be in the `AudioTranscriptionOptions` itself, so I'll go ahead and remove.

Reply:
I think the intention is for the `@sharedRoute`, but typespec not able to write the route based on different enum value.

text based route
https://github.com/Azure/azure-rest-api-specs/blob/f11aea7b86a943455b4759c70a8567b1f14d1f34/specification/cognitiveservices/OpenAI.Inference/routes/audio_transcription.tsp#L28-L34

json based route
https://github.com/Azure/azure-rest-api-specs/blob/f11aea7b86a943455b4759c70a8567b1f14d1f34/specification/cognitiveservices/OpenAI.Inference/routes/audio_transcription.tsp#L44-L50